tests/bake.go

```diff
@@ -149,6 +152,124 @@ RUN echo "Hello ${HELLO}"
 	}
 }
 
+func testBakePrintSensitive(t *testing.T, sb integration.Sandbox) {
+	testCases := []struct {
+		name string
+		f    string
+		dt   []byte
+	}{
+		{
+			"HCL",
+			"docker-bake.hcl",
+			[]byte(`
+target "build" {
+  args = {
+    HELLO = "foo"
+  }
+
+  cache-from = [
+    "type=gha",
```

Comment:
small nit, In https://github.com/docker/buildx/pull/2827/files#diff-095bae3e0563ad1f84bf1ca114f4687cc22db9bbbdd54299055015838038f235R84 I specifically sets `token` attribute in the definition to make sure it's being printed as intended if you want to add this new case.

Reply:
Good idea. Done.